bokeh/plotting_helpers.py

```diff
@@ -125,6 +125,7 @@ def _process_sequence_literals(glyphclass, kwargs, source):
         kwargs[var] = var
 
 def _make_glyph(glyphclass, kws, extra):
+        if extra is None: return None
```

Comment:
I don't think this is great code, BTW, but it's already not great code. Some day all of this should be more finely factored...